ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -97,44 +97,118 @@ if [[ "${RUN_INTEGRATION_TESTS}" == "yes" || \
 
   # Changing the PATH disables the Bazel cache, so use an absolute path.
   readonly GCLOUD="/usr/local/google-cloud-sdk/bin/gcloud"
+  readonly GCLOUD_CONFIG="cloud-cpp-integration"
+
+  cleanup_hmac_service_account() {
+    local -r ACCOUNT="$1"
+    log_normal "Deleting account ${ACCOUNT}"
+    # We can ignore errors here, sometime the account exists, but the bindings
+    # are gone (or were never created). The binding is harmless if the account
+    # is deleted.
+    "${GCLOUD}" --quiet projects remove-iam-policy-binding \
+        "${GOOGLE_CLOUD_PROJECT}" \
+        --member "serviceAccount:${ACCOUNT}" \
+        --role roles/iam.serviceAccountTokenCreator || true >/dev/null
+    "${GCLOUD}" --quiet iam service-accounts delete --quiet \
```

Comment:
The second `--quiet` is was just my mistake.

Strangely, `--quiet` is what I want, it means:

```
     --quiet, -q
        Disable all interactive prompts when running gcloud commands. If input
        is required, defaults will be used, or an error will be raised.
        Overrides the default core/disable_prompts property value for this
        command invocation. This is equivalent to setting the environment
        variable CLOUDSDK_CORE_DISABLE_PROMPTS to 1.
```

It might be possible to replace `>/dev/null` with a suitable application of `--no-user-output-enabled` and with `--verbosity=none` or something, but I think `>/dev/null` is easier to read.